src/main/java/io/leonis/torch/Torch.java

```diff
@@ -0,0 +1,63 @@
+package io.leonis.torch;
+
+import com.googlecode.lanterna.TextColor;
+import com.googlecode.lanterna.gui2.*;
+import com.googlecode.lanterna.screen.Screen;
+import com.googlecode.lanterna.screen.TerminalScreen;
+import com.googlecode.lanterna.screen.VirtualScreen;
+import com.googlecode.lanterna.terminal.DefaultTerminalFactory;
+import com.googlecode.lanterna.terminal.Terminal;
+import io.leonis.torch.component.TextBackground;
+import io.leonis.torch.component.graph.Gradient;
+import io.leonis.torch.component.graph.LineGraph;
```

Comment:
Remind me to provide you with the correct settings for intellij in slack. We decided to use a wildcard import if more than 1 dependency from the same package. (e.g. all `io.leonis.torch.component.graph.<specific>` imports should be changed to `io.leonis.torch.component.graph.*`). Luckily this is a setting and we can fix this during the final formatting.